### ISSUE TITLE
The link hint some time use default

### ISSUE THREAD

**Issue by binhtddev** [dracula/vimium#12]
I don't known why but link hint on my browser sometimes uses the default theme.
I must use the more specific css matching from other vimium theme to solve this problem.
This is my change:
```
/* Link hint boxes */
#vimiumHintMarkerContainer div.internalVimiumHintMarker, #vimiumHintMarkerContainer div.vimiumHintMarker {
  background: #282a36;
  font-size: 12px;
  border: 0.25em solid #ffb86c;
  border-radius: 0.34em;
  box-shadow: 0em 0.1em 0.6em 0.1em rgba(0, 0, 0, 0.4);
}

/* Link hint text */
#vimiumHintMarkerContainer div span {
  color: #f8f8f2;
  font-size: inherit;
  text-shadow: none;
}

/* Link hint matching characters */
#vimiumHintMarkerContainer div > .matchingCharacter ~ span {
  color: #6272a4;
}
```

**Comment by Trollwut** [comment]
Hey mate!

Thanks for your contributions! :)

Can you give me an example where it falls back to default?

Also in your code, if you write a letter, it would be better to color the *written* letter instead of the remaining ones. I think this is more intuitively for the user to distinct what he has pressed.

**Comment by binhtddev** [comment]
This is the default link hint, it occur when I open a browser (Brave):
![image](https://user-images.githubusercontent.com/26183671/156867989-0993f159-2677-4a9f-80be-88282850fb46.png)
If I refresh the page many times, it sometimes uses the theme correctly like this:
![image](https://user-images.githubusercontent.com/26183671/156868244-ed3fccff-a841-4a46-8e85-3435bc5348c9.png)
This means dracula theme css not working at all (May be it is outdated)

**Comment by Trollwut** [comment]
Indeed. It seems like on first launch into a site, the default style is being used. Upon refreshing or clicking another page in this site, the custom CSS is being used.

I can reproduce that behaviour.
